src/patternfly/components/Table/TableCells/table-cell-check.hbs

```diff
@@ -2,10 +2,10 @@
 {{!-- TODO: automate `td/th` based up `IsThead/IsTbody` and make overridable --}}
 {{!-- TODO: automate attribution --}}
 
-{{#> table-td table-td--IsCheckbox=true}}
+{{#> table-cell table-cell--IsCheckbox=true}}
```

Comment:
Similar thing for the table-cell-toggle file, but the `th` element will need an aria-label of its own when the only content is an interactive, non-text content. "Row select" in this case and "Row expansion" for the toggle, or something like that (I think I used those in my React PR)